instrumentation/action_view/lib/opentelemetry/instrumentation/action_view/span_subscriber.rb

```diff
@@ -10,6 +10,16 @@ module ActionView
       # The SpanSubscriber is a special ActiveSupport::Notification subscription
       # handler which turns notifications into generic spans, taking care to handle
       # context appropriately.
+
+      def self.subscribe(pattern = nil, callable = nil)
```

Comment:
This is a hacky version of this call path
https://github.com/rails/rails/blob/v6.1.4.1/activesupport/lib/active_support/notifications.rb#L234-L236
https://github.com/rails/rails/blob/v6.1.4.1/activesupport/lib/active_support/notifications/fanout.rb#L24-L36

The idea is if we push ourselves to the front of the list we will avoid the issue of a caller before us raising and preventing us from starting/ending a span.  This is a similar approach to what exists currently without pushing the ordering responsibility onto the notifier.